contrib/examples/cluster.yaml

```diff
@@ -8,6 +8,7 @@ objects:
   kind: ClusterVersion
   metadata:
     name: ocp-v3-9
+    namespace: cluster-operator
```

Comment:
The namespaces for these ClusterVersions will be stripped out. To retain the namespace, it needs to come from a parameter.